configure.py

```diff
@@ -1061,6 +1064,7 @@ def find_ninja():
                 'auth/authenticator.cc',
                 'auth/common.cc',
                 'auth/default_authorizer.cc',
+                'auth/ldap_connection.cc',
```

Comment:
could you please add this source file to `auth/CMakeLists.txt` so it is compiled as well ? see also https://github.com/scylladb/scylla-enterprise/commit/ee073326f137024cb18194277acfe2d097552d0b